### PR TITLE
Edit message in chat test

### PR DESCRIPTION
**Pull request by Serhy**
Edit message in chat test (tests editing in 1-1 and public)

### CODE REVIEW

File: test/appium/tests/atomic/chats/test_one_to_one.py [hunk 1]
```diff
@@ -196,6 +196,56 @@ def test_send_non_english_message_to_newly_added_contact_on_different_networks(s
...
+        chat_name = device_1_home.get_random_chat_name()
+        device_1_home.join_public_chat(chat_name)
+        device_1_public_chat = device_1_home.get_chat_view()
+        message_before_edit = "Message BEFORE edit 1"
+        message_after_edit = "Message AFTER edit 2"
+        device_1_public_chat.send_message(message_before_edit)
+        device_1_public_chat.edit_message_in_chat(message_before_edit, message_after_edit)
+        if not device_1_public_chat.element_by_text_part("⌫ Edited").is_element_displayed():
+            self.errors.append('No mark in message bubble about this message was edited')
+
```
Comment (churik) [inline test/appium/tests/atomic/chats/test_one_to_one.py:220]: would be good to add check that you can't edit messages of other users

File: test/appium/tests/atomic/chats/test_one_to_one.py [hunk 2]
```diff
@@ -196,6 +196,56 @@ def test_send_non_english_message_to_newly_added_contact_on_different_networks(s
         #     self.errors.append("Updated profile picture is not shown in one-to-one chat")
         self.errors.verify_no_errors()
 
+    @marks.testrail_id(111111111111)
```
Comment (churik) [inline test/appium/tests/atomic/chats/test_one_to_one.py]: don't forget to change it:)

File: test/appium/views/chat_view.py [hunk 3]
```diff
@@ -603,6 +603,14 @@ def send_message(self, message: str = 'test message'):
         self.chat_message_input.send_keys(message)
         self.send_message_button.click()
 
+    def edit_message_in_chat(self, message_to_edit, message_to_update):
+        self.driver.info("**Looking for message '%s' to edit it**" % message_to_edit)
+        self.element_by_text_part(message_to_edit).long_press_element()
+        self.element_by_translation_id("edit").click()
+        self.chat_message_input.clear()
+        self.chat_message_input.send_keys(message_to_update)
```
Comment (churik) [inline test/appium/views/chat_view.py:611]: can use send_message here

Comment (Serhy) [reply test/appium/views/chat_view.py:611]: Done.

File: test/appium/tests/atomic/chats/test_one_to_one.py (hunk 2 above)
Comment (Serhy) [reply test/appium/tests/atomic/chats/test_one_to_one.py]: Thanks:) Changed.

File: test/appium/tests/atomic/chats/test_one_to_one.py (hunk 1 above)
Comment (Serhy) [reply test/appium/tests/atomic/chats/test_one_to_one.py:220]: Added.

**Review by churik — APPROVED** [review]

**Pull request merged**